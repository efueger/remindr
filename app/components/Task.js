import React from 'react';
import { StyleSheet, View, Text } from 'react-native';

const styles = StyleSheet.create({
  container: {},
});

export default function Task({ counter, increment, decrement }) {
  return (
    <View style={styles.container}>
      <Text>Lorem</Text>
    </View>
  );
}
