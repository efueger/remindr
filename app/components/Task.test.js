import React from 'react';
import Task from './Task';
import renderer from 'react-test-renderer';

it('changes the class when hovered', () => {
  const component = renderer.create(
    <Task />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
