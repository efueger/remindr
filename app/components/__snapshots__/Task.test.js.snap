exports[`test changes the class when hovered 1`] = `
<View
  style={Object {}}>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Lorem
  </Text>
</View>
`;
